### PR TITLE
Update library to Contributors library standards

### PR DESCRIPTION
**Pull request by maxdeviant**
This PR updates `purescript-css` to adhere to the [Contributors library guidelines](https://github.com/purescript-contrib/governance/blob/main/library-guidelines.md).

This is part of the https://github.com/purescript-contrib/governance/issues/13 effort.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/slamdata/purescript-css.git"
+    "url": "https://github.com/purescript-contrib/purescript-css.git"
```
Comment (maxdeviant) [inline bower.json:14]: The registry needs to be updated with the new URL. There's an open PR for this: https://github.com/purescript/registry/pull/81.

**Review by thomashoneyman — APPROVED** [review]

**Comment by maxdeviant** [comment]
Verified that `spago build` and `spago test` work as expected:

```
λ spago build
[info] Installation complete.
[info] Build succeeded.
λ spago test
[info] Installation complete.
[info] Build succeeded.
[info] Tests succeeded.
```

File: bower.json (hunk 1 above)
Comment (thomashoneyman) [reply bower.json:14]: I commented on the registry, but I believe we’re preserving the original repository locations and following redirects. So with that in mind we only need to change this to https

Comment (maxdeviant) [reply bower.json]: Okay, reverted to match the URL in the registry.

**Pull request merged**